components/table/TestResultTable.tsx

```diff
@@ -1,16 +1,18 @@
 import { useRouter } from 'next/router';
 import React, { useCallback } from 'react';
 import { useIntl } from 'react-intl';
-import { BuildingBlockTestSummary } from '../../service/types';
+import { BuildingBlockEndpointTest, BuildingBlockTestSummary } from '../../service/types';
 import TableErrorHandling from './TableErrorHandling';
 import TestResultTableHeader from './TestResultTableHeader';
 import TestResultTableRow from './TestResultTableRow';
 
 type Props = {
   bbSummary: BuildingBlockTestSummary | undefined
+  // eslint-disable-next-line no-unused-vars
```

Comment:
I don't think this line is necessary. Can you remove it?

Reply:
oh noo. I need this parameter in the func and eslint throw error because of it so I need to keep it that way

Reply:
In this case we need to modify eslint configuration by using this rule:
https://typescript-eslint.io/rules/no-unused-vars/

Let's handle it in a separate task so it doesn't block this PR